.travis.yml

```diff
@@ -1,9 +1,9 @@
 sudo: false
 language: node_js
 node_js:
-  - "8"
-  - "10"
-  - "12"
+  - '8'
+  - '10'
+  - '12'
```

Comment:
Please revert unrelated changes, or alternatively open a standalone pull request to make them.

What VSCode plugin do you use to format the YAML file? Maybe we need to tell Prettier to ignore `.travis.yml`?

Reply:
Yes, I use Prettier

Reply:
ps: `.prettierignore` doesn't work.